### PR TITLE
feat(snomed.reasoner): Reinstate equivalent concept merging extension

### PR DESCRIPTION
**Pull request by apeteri**

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.reasoner/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.reasoner/plugin.xml]: Would it be possible to use the newer plugin service registration mechanism? ClassPathScanner + service/toolingId lookup.

**Review by cmark — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.snomed.reasoner/plugin.xml (hunk 1 above)
Comment (apeteri) [reply snomed/com.b2international.snowowl.snomed.reasoner/plugin.xml]: I've replaced the extension point with `ClassPathScanner` and a registry in commit cfef3f5.

**Review by cmark — APPROVED** [review]
🥓

**Review by nagyo — APPROVED** [review]
LGTM!

**Pull request merged**